book/src/pong-tutorial/pong-tutorial-05.md

```diff
@@ -39,6 +39,7 @@ Then, we'll create `systems/winner.rs`:
 #     }
 #
 #     pub const ARENA_WIDTH: f32 = 100.0;
+#     pub const ARENA_HEIGHT: f32 = 100.0;
```

Comment:
I'm unsure if this is legit.

Reply:
it's legit :v:

Reply:
I see, makes sense. This explains some of the odd behavior I was seeing where using the code block copy button would generate non-compiling code; not sure if this is a known challenge?